packages/xstate-react/package.json

```diff
@@ -54,7 +54,7 @@
     "url": "https://github.com/statelyai/xstate/issues"
   },
   "peerDependencies": {
-    "react": "^16.8.0 || ^17.0.0 || ^18.0.0",
+    "react": "^16.8.0 || ^17.0.0 || ^18.0.0 || ^19.0.0",
```

Comment:
```suggestion
    "react": "^16.8.0 || ^17.0.0 || ^18.0.0 || ^19.0.0-0",
```

Reply:
This is a strategy used by Vercel's `styled-jsx`:
https://github.com/vercel/styled-jsx/blob/d3949b20cdcc60effcbefd91c169a7129d7f8cf8/package.json#L101-L103